src/components/services/EdgeCoreManager.js

```diff
@@ -94,32 +92,17 @@ export class EdgeCoreManager extends React.PureComponent<Props, State> {
     }
   }
 
-  hideSplash() {
-    if (!this.splashHidden) {
-      this.splashHidden = true
-      SplashScreen.close({
-        animationType: SplashScreen.animationType.fade,
-        duration: 850,
-        delay: 500
-      })
-    }
-  }
-
   onContext = (context: EdgeContext) => {
     console.log('EdgeContext opened')
     context.on('close', () => {
       console.log('EdgeContext closed')
       this.setState({ context: null })
     })
-    this.setState(
-      state => ({ context, counter: state.counter + 1 }),
-      () => this.hideSplash()
-    )
+    this.setState(state => ({ context, counter: state.counter + 1 }))
   }
 
   onError = (error: Error) => {
     console.log('EdgeContext failed', error)
-    this.hideSplash()
```

Comment:
Where do we animate the app's opacity, so everything is transparent until the core first loads, and then fades in to the login screen?

Reply:
On iOS we don't need this anymore. The new storyboard combined with the changes I made to the AppDelegate file makes this unnecessary.

https://reactnative.dev/docs/publishing-to-app-store#pro-tips